client/components/GenericTable/GenericTable.tsx

```diff
@@ -104,28 +91,18 @@ const GenericTable: {
 				</Tile>
 			) : (
 				<>
-					<Box mi='neg-x24' pi='x24' flexShrink={1} flexGrow={1} ref={ref} overflow='hidden'>
-						<ScrollableContentWrapper overflowX>
-							<Table fixed={fixed} sticky>
-								{header && (
-									<Table.Head>
-										<Table.Row>{header}</Table.Row>
-									</Table.Head>
-								)}
-								<Table.Body>
-									{RenderRow &&
-										(results ? (
-											results.map((props, index) => (
-												<RenderRow key={props._id || index} {...props} />
-											))
-										) : (
-											<Loading />
-										))}
-									{children && (results ? results.map(children) : <Loading />)}
-								</Table.Body>
-							</Table>
-						</ScrollableContentWrapper>
-					</Box>
+					<GenericTableV2 fixed={fixed} ref={ref}>
+						{header && <GenericTableHeader>{header}</GenericTableHeader>}
+						<GenericTableBody>
+							{!results && <GenericTableLoadingTable headerCells={headerCells} />}
```

Comment:
Can we break this with isLoading method to clarify the intention of this block and make it easy to read?